tests/models/fastspeech2_conformer/test_modeling_fastspeech2_conformer.py

```diff
@@ -408,7 +408,7 @@ def test_training_integration(self):
         batch_size, max_text_len = input_ids.shape
         pitch_labels = torch.rand((batch_size, max_text_len, 1), dtype=torch.float, device=torch_device)
         energy_labels = torch.rand((batch_size, max_text_len, 1), dtype=torch.float, device=torch_device)
-        duration_labels = torch.normal(10, 2, size=(batch_size, max_text_len)).clamp(1, 20).int()
+        duration_labels = torch.normal(10, 2, size=(batch_size, max_text_len)).clamp(1, 20).int().to(torch_device)
```

Comment:
Is there a reason for not passing `device` directly when it's created? It should be quicker

```suggestion
        duration_labels = torch.normal(10, 2, size=(batch_size, max_text_len), device=torch.device).clamp(1, 20).int()
```